### PR TITLE
Fix #3354 by pulling VSC::beresp_bodybytes out of the lowest VFP.

### LINKED ISSUE #3354: no Backend Counters beresp_bodybytes in varnishstat if option set beresp.do_gzip = true;
## Current Behavior

if i use the option "set beresp.do_gzip = true;" in "vcl_backend_response", I no longer have the value of Backend Counters beresp_bodybytes in varnishstat.
```
# with option: varnishstat -1|grep VBE.boot.front_fd.be 
VBE.boot.front_fd.bereq_hdrbytes                         71549599706     51664.65 Request header bytes
VBE.boot.front_fd.bereq_bodybytes                         2573041705      1857.95 Request body bytes
VBE.boot.front_fd.beresp_hdrbytes                        22571311577     16298.33 Response header bytes
VBE.boot.front_fd.beresp_bodybytes                                 0         0.00 Response body bytes
```

## Steps to Reproduce (for bugs)

1.add set beresp.do_gzip = true; in vcl_backend_response
2.request on varnish
3.varnishstat -1|grep VBE.boot.front_fd.be 
4.VBE.boot.front_fd.beresp_bodybytes                                 0         0.00 Response body bytes

## Context

we lost our volume production indicator by data by backend

## Your Environment

* Version used: varnish-6.3.1 revision 6e96ff048692235e64565211a38c41432a26c055
* Operating System and version:ubuntu xenial
* Source of binary packages used (if any)

### PR DESCRIPTION
**Pull request by bsdphk**
subject says it all really...

### CODE REVIEW

**Comment by bsdphk** [comment]
I lean that this should not be merged before the 2020-09-15 release.

**Comment by bsdphk** [comment]
Hmm this ended up as a git-f**kup, look at the diff, I'll commit a clean version when the time comes.

**Review by Dridi — APPROVED** [review]

File: bin/varnishd/cache/cache_fetch_proc.c [hunk 1]
```diff
@@ -117,18 +117,21 @@ VFP_Setup(struct vfp_ctx *vc, struct worker *wrk)
...
 {
 	struct vfp_entry *vfe, *tmp;
+	size_t rv = 0;
 
 	VTAILQ_FOREACH_SAFE(vfe, &vc->vfp, list, tmp) {
 		if (vfe->vfp->fini != NULL)
 			vfe->vfp->fini(vc, vfe);
 		VSLb(vc->wrk->vsl, SLT_VfpAcct, "%s %ju %ju", vfe->vfp->name,
 		    (uintmax_t)vfe->calls, (uintmax_t)vfe->bytes_out);
+		rv = vfe->bytes_out;
```
Comment (nigoroll) [inline bin/varnishd/cache/cache_fetch_proc.c:131]: (minor) could move this one line up and then use `rv` in the `VSLb()`.

**Review by nigoroll — APPROVED** [review]

**Comment by bsdphk** [comment]
Obsoleted, by cleaned up commit.

@nigoroll's suggestion included.

**Pull request closed without merging**